src/clients.test.ts

```
@@ -1,29 +1,41 @@
+  expect(getClient('bearBelts')).toBeTruthy();
 });
 
-export {}
+test('throws a descriptive error for an unknown client key', async () => {
+  const { getClient } = await import('./clients');
+  const invalidClientKey = 'unknown-brand' as import('./services').BrandKey;
 
+  expect(() => getClient(invalidClientKey)).toThrow('Unknown Shopify client key: unknown-brand');
+});
+
+export {}
```

Biome (2.4.6)

[error] Do not export from a test file.

(lint/suspicious/noExportsInTest)